src/program/lwaftr/tests/propbased/common.lua

```diff
@@ -0,0 +1,52 @@
+module(..., package.seeall)
+
+-- common definitions for property-based tests for snabb config
+
+local S = require("syscall")
+local pci = require("lib.hardware.pci")
+
+function make_handle_prop_args(name, duration, pidbox)
+  local handler = function(prop_args)
+     if #prop_args ~= 1 then
+        print("Usage: snabb quickcheck prop_sameval PCI_ADDR")
+        os.exit(1)
+     end
+  
+     -- TODO: validate the address
+     local pci_addr = prop_args[1]
```

Comment:
Does this code already sufficiently address validation? If so, the TODO comment should be removed, I'd say.